news/2019-08-20-new-featured-artist-billain.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title: "New Featured Artist: Billain"
+date: 2019-08-20 04:00:00 +0000
+---
+
+[**Billain**](https://osu.ppy.sh/beatmaps/artists/46) descends onto the scene as osu!'s latest Featured Artist, complete with a selection of tracks from his latest album, *Nomad's Revenge*!
+
+![](https://assets.ppy.sh/artists/46/header.jpg)
+
+Made famous in osu! by the giant collaborative beatmap [**Boogie**](https://osu.ppy.sh/beatmapsets/411435#osu/966177) hosted by [Spaghetti](https://osu.ppy.sh/users/4761930) back in 2016, [**Billain**](https://osu.ppy.sh/beatmaps/artists/46) has been an eminent name among much of the high-end mapping scene ever since.
+
+His unique fusion of atmospheric, neurofunk-inspired sound design interwoven with classic drum & bass style leads produces a uniquely compelling long-form experience that we're sure people are bound to pick up and experiment with.
+
+Some people have historically turned their brain off when confronted with tracks longer than two minutes long - but really, give them a shot! There's a reason almost all of the notable beatmaps people remember fondly are all pretty long ones.
+
+We've managed to nab a selection of **18** tracks from his widely varied library, including five tracks from his latest release with *Renraku Global Media*, [*Nomad's Revenge*](https://renraku.bandcamp.com/album/nomads-revenge).
+
+Check it out:
```

Comment:
```suggestion
Check them out:
```
or maybe "Check some of them out:" ? because "it"/"them" makes it sound like these are from nomad's revenge, but not all of them are